packages/shared/src/hooks/useFeed.ts

```diff
@@ -155,6 +159,7 @@ export default function useFeed<T>(
     enabled: query && tokenRefreshed,
     refetchOnReconnect: false,
     refetchOnWindowFocus: false,
+    retry: shouldRetry ? false : 3,
```

Comment:
Force enrolled to never load basically.

Reply:
Not sure I understand the idea of this. Basically if it's a custom feed and the user is enrolled, `shouldRetry` will be true but we set the `retry` itself to be false. Can you elaborate please?

Reply:
The idea is that when custom feed, user is enrolled and not plus they never see a feed.
Currently it will retry 3 times before showing empty screen, but we know by hand 1 retry is enough as it's always empty.

Reply:
But you don't check if the have plus, only if enrolled. And also variable `shouldRetry` which indicates if you shouldn't retry is confusing

Reply:
Adjusted now let me know what you think.
I initially thought of just not firing query, but since it's so generic it became super messy.

Reply:
I don't see that it was changed